app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  # Callback
+  after_save :updates_post_likes_counter
+
+  # Methods
+  def updates_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] It will be nice to make the `updates_post_likes_counter` method private since we only call it inside the class.
```ruby
  private

  def updates_post_likes_counter
    post.update(likes_counter: post.likes.count)
  end
```

Reply:
- Method made private.